seshdash/utils/model_tools.py

```diff
@@ -130,3 +133,24 @@ def get_status_card_items(site):
     except Exception  ,e :
         looger.error(e)
         pass
+
+def get_site_measurements(site):
+
+    site_measurements = site.site_measurements
+    # getting all measurement fields
+    site_measurements_fields = site_measurements._meta.get_fields()
+    site_measurements_items = []
+    for field in site_measurements_fields:
+        site_measurements_items.append(getattr(site_measurements, field.name))
+
+    #removing non char items
+    for i,item in enumerate(site_measurements_items):
```

Comment:
[<Sesh_Site: Mukuyu HC>, 2, u'soc', u'trans', u'AC_output_absolute', u'AC_Load_in', u'AC_Load_out', u'AC_Voltage_in', u'AC_Voltage_out', u'AC_input', u'AC_output', u'pv_production', u'genset_state', u'main_on', u'relay_state', u'trans', u'AC_input']
it print something like this 
needed to pop up out some of them